app/models/saved_claim/coe_claim.rb

```diff
@@ -95,10 +95,26 @@ def relevant_prior_loans(form_copy)
 
   def periods_of_service(form_copy)
     parsed_form['periodsOfService'].each do |service_info|
+      # values from the FE for military_branch are:
+      # ["Air Force", "Air Force Reserve", "Air National Guard", "Army", "Army National Guard", "Army Reserve",
+      # "Coast Guard", "Coast Guard Reserve", "Marine Corps", "Marine Corps Reserve", "Navy", "Navy Reserve"]
+      # these need to be formatted because LGY only accepts [ARMY, NAVY, MARINES, AIR_FORCE, COAST_GUARD, OTHER]
+      # and then we have to pass in ACTIVE_DUTY or RESERVE_NATIONAL_GUARD for service_type
+      military_branch = service_info['militaryBranch'].parameterize.underscore.upcase
+      service_type = 'ACTIVE_DUTY'
+
+      if military_branch.include?('NATIONAL_GUARD') || military_branch.include?('RESERVE')
```

Comment:
```suggestion
      if ['RESERVE', 'NATIONAL_GUARD'].include?(military_branch)
```

This may be. a cleaner way to write the conditional

Reply:
Thanks Lindsey!  I made the suggested change but then rubocop suggested I use %w instead of string literals.  Thank you for noticing - that does look cleaner.

Reply:
Actually Lindsey, I had to change the logic again... the military branch is the longer string (ex. AIR_NATIONAL_GUARD) so the .include? fails when we had it that way initially.  I made some changes - let me know what you think.  Thanks!